kafka-connect-gcp-storage/src/main/scala/io/lenses/streamreactor/connect/gcp/storage/storage/GCPStorageDirectoryLister.scala

```diff
@@ -63,6 +62,8 @@ class GCPStorageDirectoryLister(connectorTaskId: ConnectorTaskId, storage: Stora
           )
         }
 
+      logger.trace(s"[$connectorTaskId] Searching directory $prefix for $recurseLevels, found ${foundResults.size}")
```

Comment:
If we don't leave trace logs this can be removed

Reply:
Logging can be configured by the user to display the level of logging they require.

It's useful to have debug or trace logging for diagnosis purposes.

This is a bit too verbose for debug so I used trace.

[Some more information about configuring Kafka Connect logging here](https://docs.lenses.io/5.4/support/connectors/kafka-connect-logging/#3e1abdf61cd87e0067222c933984bb8b).

Reply:
Okay, just wanted to know as sometimes this is configurable, in other projects I seen it being dropped after development (or ascended into `INFO` level)